client/dogs/FormPage.js

```diff
@@ -296,6 +297,7 @@ export type InitialProps = {|
 
 export type Props = {
   ...InitialProps,
+  children: Element<any>,
```

Comment:
Do you need this? You're not using `this.props.children` below as far as I can tell.